File: selling-used-items-app-frontend/src/app/model/comment.cs
export class Comment {
  id: number;
  creatorId: number;
  targetUserId: number;
  advertisementId: number;
  message: string;
  rating: number;
  isApproved: boolean;

  constructor(
    id: number,
    creatorId: number,
    targetUserId: number,
    advertisementId: number,
    message: string,
    rating: number,
    isApproved: boolean
  ) {
    this.id = id;
    this.creatorId = creatorId;
    this.targetUserId = targetUserId;
    this.advertisementId = advertisementId;
    this.message = message;
    this.rating = rating;
    this.isApproved = isApproved;
  }
}

File: selling-used-items-app-frontend/src/app/model/comment.ts
export class Comment {
    id: number = 0;
    creatorId: number = 0;
    purchaseId: number = 0;
    targetUserId: number = 0;
    message: string = '';
    rating: number = 0;
    isApproved: boolean = false;
  
    constructor(
      id: number = 0,
      creatorId: number = 0,
      targetUserId: number = 0,
      advertisementId: number = 0,
      message: string = '',
      rating: number = 0,
      isApproved: boolean = false
    ) {
      this.id = id;
      this.creatorId = creatorId;
      this.targetUserId = targetUserId;
      this.message = message;
      this.rating = rating;
      this.isApproved = isApproved;
    }
  }
  